mypy/semanal.py

```diff
@@ -2241,7 +2247,8 @@ def process_module_assignment(self, lvals: List[Lvalue], rval: Expression,
             # extra elements, so no error will be raised here; mypy will later complain
             # about the length mismatch in type-checking.
             elementwise_assignments = zip(rval.items, *[v.items for v in seq_lvals])
-            for rv, *lvs in elementwise_assignments:
+            for part in elementwise_assignments:
+                rv, lvs = part[0], list(part[1:])
```

Comment:
Maybe add a TODO here to rewrite this back to non-ugly form when this will be implemented in mypyc?